### PR TITLE
DOCS: fix missing argument for toTitlecase()

### PR DESCRIPTION
**Pull request by thadguidry**

### CODE REVIEW

**Comment by magdmartin** [comment]
looks good to me. If there is no objection, I can merge this early next week.

**Review by tfmorris — COMMENTED** [review]
My apologies! I started a review of this and never clicked "Submit review." The parameter naming is a nit, but the example is wrong and needs to be fixed.

File: docs/manual/grelfunctions.md [hunk 1]
```diff
@@ -86,10 +86,12 @@ Returns string s converted to all lowercase characters.
 
 Returns string s converted to all uppercase characters.
 
-###### toTitlecase(s) {#totitlecases}
+###### toTitlecase(s, delimiter (optional)) {#totitlecases}
 
 Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, `"Once upon a midnight DREARY".toTitlecase()` returns the string “Once Upon A Midnight Dreary”.
 
+Optionally, a string of delimiter characters used to separate words. The first character and the first non-delimiter character after a delimiter will be captialized. For example, `"jean-michel jarre".toTitlecase(" -")` returns "Jean Michel Jarre".
```
[the remark this answers is not in the document]
Comment (thadguidry) [reply docs/manual/grelfunctions.md:93]: Thanks @tfmorris fixed in #380 awaiting your review

File: docs/manual/grelfunctions.md [hunk 2]
```diff
@@ -86,10 +86,12 @@ Returns string s converted to all lowercase characters.
 
 Returns string s converted to all uppercase characters.
 
-###### toTitlecase(s) {#totitlecases}
+###### toTitlecase(s, delimiter (optional)) {#totitlecases}
```
[the remark this answers is not in the document]
Comment (thadguidry) [reply docs/manual/grelfunctions.md:89]: fixed in #380

**Pull request merged**